src/main/plugins/org.dita.pdf2/build_template.xml

```diff
@@ -242,7 +243,15 @@ with those set forth herein.
     <property name="args.tablelink.style" value="NUMTITLE"/>
   </target>
 
-  <target name="transform.topic2fo.index">
+  <target name="transform.topic2fo.index.init">
+    <condition property="_pdf2.index.skip" value="true">
+      <istrue value="${pdf2.index.skip}"/>
+    </condition>
+  </target>
+
+  <target name="transform.topic2fo.index"
+          depends="transform.topic2fo.index.init"
+          unless="_pdf2.index.skip">
```

Comment:
Added `_pdf2.index.skip` property so we can accept truthy value for the skip property. This way the value can be falsy and the target will not be skipped.